README.md

```diff
@@ -118,12 +118,20 @@ Note on NVME storage: Samsung PM981 drives will not work out of the box if at al
 | SSDT-PTSWAK           | Fix sleep issues               |
 | SSDT-SBUS-MCHC        | SBUS fix                       |
 | SSDT-USBX             | USBX patch                     |
+| SSDT-BKey             | Fix display brightness keys    |
 
-## Pre-Install & BIOS settings
 
-First, read the [Dortania OC guide](https://dortania.github.io/OpenCore-Install-Guide/). The guide will take you through the creation of installation USB and drive formatting. Update to the latest firmware (the easiest way is to use fwupd on linux).
+## Pre-Install: Creating the installation USB stick
 
-Then do the following BIOS settings:
+First, read the [Dortania OC guide](https://dortania.github.io/OpenCore-Install-Guide/). The guide will take you through the creation of installation USB and drive formatting. Update the laptop to the latest BIOS firmware from your existing OS (the easiest way is to use fwupd on linux). Once using MacOS, it's a bit more complicated to update the BIOS.
+
+
+## Getting a valid Mac serial key
+
+- [Fix iServices](https://dortania.github.io/OpenCore-Post-Install/universal/iservices.html#generate-a-new-serial) if you want to use iMessage, FaceTime, iCloud. You need a valid, unique Mac serial key (this config.plist does not have one) to be able to use Apple's cloud services and authentication. If you don't, you won't be able to login with an Apple ID, thus no App Store either! Follow the link to generate a serial and update directly the config.plist. We use the SMBIOS *MacBookPro15,4* as it is the closest Mac to our internal hardware.
```

Comment:
I think App Store can be used without a valid serial, you just log in inside the app itself? FaceTime, iMessage and iCloud Drive require a valid serial for sure though.

Reply:
Doesn't work. You need the Apple ID valid before you can install anything.